src/test/common/page/page.js

```diff
@@ -85,9 +85,11 @@ class Page {
     }
   }
 
-  async findAllByClassName (className) {
+  async findAllByClassName (className, waitFirst = true) {
```

Comment:
Are there any situations where we actually want this to wait for elements to exist? I guess so as its already in the code, but just asking the question.

Reply:
Actually, there aren't, I've removed the waiting from both the findAll methods as it isn't needed and actually gets in the way.